go/cmd/vtctldclient/command/root.go

```diff
@@ -204,14 +203,6 @@ func getClientForCommand(cmd *cobra.Command) (vtctldclient.VtctldClient, error)
 		}
 		onTerm = append(onTerm, ts.Close)
 
-		// Use internal vtctld server implementation.
-		// Register a nil grpc handler -- we will not use tmclient at all but
-		// a factory still needs to be registered.
```

Comment:
Was this comment just incorrect?

Reply:
It *was* correct. It is no longer. Now we do make some tmclient calls and register the grpc handler separately.